bin/cml/tensorboard/connect.js

```diff
@@ -81,6 +81,11 @@ exports.command = 'connect';
 exports.description = `${DESCRIPTION}\n${DOCSURL}`;
 
 exports.handler = async (opts) => {
+  if (new Date() > new Date('2024-01-01')) {
+    logger.error('TensorBoard.dev has been shut down as of January 1, 2024');
+    return;
+  }
```

Comment:
[TensorBoard.dev has been shut down as of January 1, 2024](https://tensorboard.dev); yes, they don't exist anymore, rendering this command pointless.